clients/client-py/taskcluster/generated/aio/auth.py

```diff
@@ -447,7 +447,7 @@ class Auth(AsyncBaseClient):
         initial team configured for this component. Contact a Sentry admin
         to have the project transferred to a team you have access to if needed
 
-        This method is ``deprecated``
+        This method is ``stable``
```

Comment:
why do we resurrect it?

Reply:
At one point we were going to stop supporting sentry within the platform, when we were using stackdriver for error reporting.  But we reversed course and now report errors directly to sentry.  And, workers use this method to get sentry credentials to report their own errors.  So basically we are no longer planning to remove this method.